gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/proc/LaunchDagProcTest.java

```diff
@@ -27,6 +27,7 @@
 import org.mockito.Mockito;
```

Comment:
maybe you can take reference from DagManagementTaskStreamImplTest 's unit test and use mockito.verify to add a unit test